tests/v2/actions/observability/stackstate.go

```diff
@@ -0,0 +1,212 @@
+package observability
```

Comment:
installing the CRDs are usually included in the actions/chart folder, why do we need a new folder for this with stackstate?

Reply:
This is not for the StackState agent chart but for the ui-plugin extensions. Since it's not a chart, I added a new folder for these extensions to keep the structure separate from chart actions.